plugins/hadoop-authorization-sentry-default/hadoop-authorization-sentry/src/main/java/com/thinkbiganalytics/datalake/authorization/config/SentryConnection.java

```diff
@@ -36,6 +37,14 @@
     private DataSource dataSource;
     private String hadoopConfiguration;
     private String sentryGroups;
+    private String authorizationGroupType;
+    private String linuxGroupFilePath;
+    private String ldapURL;
+    private String ldapAdmin;
+    private String ldapAdminPassword;
+    private LdapContextSource LdapContextSource;
+    private String LdapGroupDnPattern;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this field "LdapGroupDnPattern" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00116)